Add CTO block

alex-novak opened the pull request:
https://trello.com/c/eqI9Ru4n

albertvolkman commented:
@alex-novak the images you have here are all at least twice as large as they need to be. It looks like they only need to be 291 x 291px? I've rebased this branch, so you may need to delete your local copy.

alex-novak commented:
> @alex-novak the images you have here are all at least twice as large as they need to be. It looks like they only need to be 291 x 291px? I've rebased this branch, so you may need to delete your local copy.

fixed
